src/app/services/navigation/navigation.service.ts

```diff
@@ -30,21 +30,21 @@ export class NavigationService {
       type: MenuItemType.Link,
       tooltip: T('Storage'),
       icon: 'dns',
-      state: 'storage',
+      state: 'storage2',
     },
     {
-      name: T('Storage 2.0 (WIP)'),
+      name: T('Datasets'),
       type: MenuItemType.Link,
-      tooltip: T('Storage'),
+      tooltip: T('Datasets'),
       icon: 'dns',
-      state: 'storage2',
+      state: 'datasets',
     },
     {
-      name: T('Datasets (WIP)'),
+      name: T('S̶t̶o̶r̶a̶g̶e̶ (deprecated)'),
```

Comment:
Don't think we should cut through the word. Just typing `Deprecated` should be enough.

Reply:
It's from ticket description. I don't mind either way.